src/main/java/ua/su/service/FindSomethingByCriteria.java

```diff
@@ -0,0 +1,9 @@
+package ua.su.service;
+
+import java.util.List;
+
+public interface FindSomethingByCriteria {
+
+    List<String> findSomethingByCriteriaOrderedAsc(Integer n);
```

Comment:
The method name should be named what it does. "something" means that it returns something random. If you will do it, you should not provide comments

Reply:
Got it, the method was renamed to findClinicAddressesWhereInsuranceNotSupportedAndDoctorsNumberLessThanNOrderedAsc()